app/src/main/res/layout/activity_registration.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Changed IDs to prefix_element as in XML standard. Make sure to change it in your next push.